pkg/backend/display/object_diff.go

```diff
@@ -1268,3 +1274,14 @@ func (p *propertyPrinter) translateYAMLValue(v interface{}) (interface{}, bool)
 		return v, true
 	}
 }
+
+// if string exceeds three lines, truncate and add "..."
+func (p *propertyPrinter) truncatePropertyString(propertyString string) string {
+	const contextLines = 3
+
+	lines := strings.Split(propertyString, "\n")
+	if len(lines) > contextLines {
+		return strings.Join(lines[:3], "\n") + "..."
```

Comment:
I think this would look better on a new line, but it's up to you.
```suggestion
		return strings.Join(lines[:3], "\n") + "\n..."
```